src/java.base/share/classes/sun/launcher/resources/launcher.properties

```
@@ -226,6 +226,19 @@ The following options are macOS specific:\n\
 \    -Xdock:icon=<path to icon file>\n\
 \                      override default icon displayed in dock\n\n
 
+java.launcher.bad.option=\
+\n\
+Unrecognized showSettings option: {0}.\n\
+    Valid values are \"all\", \"locale\", \"properties\",\"security\",\
+    \"system\"(Linux only), \"vm\".\n\
```

Nit:  space after commas missing.
```suggestion
    Valid values are \"all\", \"locale\", \"properties\", \"security\", \
    \"system\"(Linux only), \"vm\".\n\
```